kalite/distributed/static/js/distributed/distributed-server.js

```diff
@@ -35,14 +35,14 @@ function show_api_messages(messages) {
     }
 }
 
-function force_sync() {
+function force_sync(zone_id, device_id) {
     // Simple function that calls the API endpoint to force a data sync,
     //   then shows a message for success/failure
     doRequest(window.Urls.api_force_sync())
         .success(function() {
             var msg = gettext("Successfully launched data syncing job.") + " ";
             msg += sprintf(gettext("After syncing completes, visit the <a href='%(devman_url)s'>device management page</a> to view results."), {
-                devman_url: window.sessionModel.get("LOCAL_DEVICE_MANAGEMENT_URL")
+                devman_url: Urls.device_management(zone_id, device_id)
```

Comment:
Identifier 'devman_url' is not in camel case.<br>Identifier 'device_management' is not in camel case.<br>Identifier 'zone_id' is not in camel case.<br>Identifier 'device_id' is not in camel case.